src/script/broadcast/BroadcastService.ts

```diff
@@ -20,11 +20,18 @@
 import {UserClients} from '@wireapp/api-client/dist/conversation';
 import {BackendClient} from '../service/BackendClient';
 
-// Broadcast service for all broadcast calls to the backend REST API.
+export interface BroadcastPayload {
+  /** Map with per-recipient data */
```

Comment:
Comment can be removed and `recipients` can be typed with: https://github.com/wireapp/wire-web-packages/blob/master/packages/api-client/src/conversation/OTRRecipients.ts